src/openforms/analytics_tools/tests/test_cspsettings_identifier.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.admin.options import get_content_type_for_model
+from django.test import TestCase, override_settings
+
+from openforms.config.models import CSPSetting
+
+from ..constants import AnalyticsTools
+from .mixin import AnalyticsMixin
+
+
+@override_settings(SOLO_CACHE=None)
+class CSPIdentifierTests(AnalyticsMixin, TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.gtm_code = "GTM-XXXX"
+        cls.ga_code = "UA-XXXXX-Y"
+        cls.matomo_url = "https://example.com"
```

Comment:
can you remove this indirection and set the values directly in the test itself? This is an unnecessary context switch when debugging/reading the test.

Reply:
I was probably thinking about adding multiple tests at first